static/src/javascripts/projects/commercial/modules/messenger/dfp-origin.js

```diff
@@ -0,0 +1,6 @@
+// @flow
+
+/* eslint-disable */
```

Comment:
Can we disable only the default export rule, to make the intention clearer?

/* eslint-disable no-default-export */